terraform/modules/hsm/main.tf

```
@@ -38,3 +61,16 @@ resource "aws_cloudhsm_v2_hsm" "cloudhsm_v2_hsm" {
   subnet_id  = "${aws_cloudhsm_v2_cluster.cluster.subnet_ids[0]}"
   cluster_id = "${aws_cloudhsm_v2_cluster.cluster.cluster_id}"
 }
+
+module "lambda_splunk_forwarder" {
+  source = "git::https://github.com/alphagov/gsp-terraform-ignition//modules/lambda_splunk_forwarder"
+
+  enabled                   = "${var.splunk}"
+  name                      = "${var.cluster_name}_hsm"
```

looks like this would mean lambda named "staging_staging_hsm" instead of "staging_hsm" due to the cluster_name being passed in... is that expected? or should it be:

```suggestion
  name                      = "hsm"
```